graylog2-server/pom.xml

```diff
@@ -359,8 +360,76 @@
     </dependencies>
 
     <build>
+        <resources>
+            <resource>
+                <directory>${webInterface.path}/build</directory>
+                <targetPath>web-interface/assets</targetPath>
+            </resource>
+            <resource>
+                <directory>src/main/resources</directory>
+            </resource>
+        </resources>
         <plugins>
             <plugin>
+                <groupId>com.github.eirslett</groupId>
```

Comment:
Question: This is only used for the production/package builds, right? Development would still use npm directly?